Pipfile.lock

```diff
@@ -0,0 +1,717 @@
+{
+    "_meta": {
+        "hash": {
```

Comment:
I think you should remove the lock file
Lock files do not make sense for libraries: you want your CI to be as close as possible to the user experience, IE pip file.